handbook/product/tracking_issues.md

```diff
@@ -0,0 +1,88 @@
+# Tracking issues
+
+## Table of contents
+
+1. [What is a Tracking issue?](#what-is-a-tracking-issue)
+1. [Creating a Tracking issue](#creating-a-tracking-issue)
+1. [Populating and maintaining a Tracking issue](#populating-and-maintaining-a-tracking-issue)
+1. [Planning a milestone with a Tracking issue](#planning-a-milestone-with-a-tracking-issue)
+1. [Using a Tracking issue for progress check-ins](#using-a-tracking-issue-for-progress-check-ins)
+1. [Contributing to the Tracking issue tool](#contributing-to-the-tracking-issue-tool)
+
+## What is a Tracking issue?
```

Comment:
I don't think "Tracking" is a proper noun, so it doesn't need to be capitalized unless it is at the beginning of a sentence. Here and elsewhere

```suggestion
## What is a tracking issue?
```